### ISSUE TITLE
Opening storage failed lock DB directory: resource temporarily unavailable

### ISSUE THREAD

**Issue by Trivikramreddy** [prometheus/prometheus#4761]
## Proposal
**Use case. Why is this important?**

*“Nice to have” is not a good use case. :)*

## Bug Report
**What did you do?**

**What did you expect to see?**

**What did you see instead? Under which circumstances?**

**Environment**

* System information:

	insert output of `uname -srm` here

* Prometheus version:

	insert output of `prometheus --version` here

* Alertmanager version:

	insert output of `alertmanager --version` here (if relevant to the issue)

* Prometheus configuration file:
```
insert configuration here
```

* Alertmanager configuration file:
```
insert configuration here (if relevant to the issue)
```

* Logs:
```
insert Prometheus and Alertmanager logs relevant to the issue here
```

**Comment by Trivikramreddy** [comment]
I'm new to prometheus. This is the error i see in the logs. prometheus is running fine but want to know what does it mean by that error. Any help on this is appreciated. Thanks !!

**Comment by simonpasquier** [comment]
It means that Prometheus can't lock access to the data directory. Either

* Prometheus doesn't permissions to write this directory.
* Or another process is running in parallel and already acquired the lock.
* Or you're using a file system (such as NFS) which isn't fully supported by Prometheus.

I'm closing it for now. If you have further questions, please use our [user mailing list](https://groups.google.com/forum/#!forum/prometheus-users), which you can also search.

**Comment by carlosqba** [comment]
I have added the recipe and the necessary scripts to update the prometheus configuration file.
In prometheus/update_prometheus_config/update_prometheus_config.py I have placed the 'temporary' function (get_data_test ()) that returns a dictionary with the necessary information to test the update of the prometheus configuration file.
Once we have the api running, we eliminate that function and call the correct one (get_data)

You can [see the changes](https://github.com/jetshop-shimmercat/ansible-jetshop-shimmercat/pull/41/commits/5cacd103987669f68d462d4116c5110873bb89df)
